part2/part2_project/src/pages/FolderPage.jsx

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useRef, useState } from 'react';
+import { forUser1, TasteUser1, getUserList } from '../util/api.js';
+import Nav from '../Components/sharing/Nav';
+import Footer from '../Components/sharing/Footer';
+import styled from 'styled-components';
+import HeaderWithInPut from '../Components/folderPage/HeaderWithInput';
+import FolderPageMain from '../Components/folderPage/FolderPageMain.jsx';
+import EditModal from '../Components/sharing/Modals/EditModal.jsx';
+import AddFolderModal from '../Components/sharing/Modals/AddFolderModal.jsx';
+import DeleteFoderModal from '../Components/sharing/Modals/DeleteFolderModal.jsx';
+import ShareModal from '../Components/sharing/Modals/ShareModal.jsx';
+import AddLinkModal from '../Components/sharing/Modals/AddLinkModal.jsx';
+import '../css/index.css';
+
+const ForFolderNav = styled(Nav)`
+  position: static;
+`;
+
+export default function FolderPage() {
+  const [inputValue, setInputValue] = useState('');
+  const [search, setSearch] = useState('');
+  const [userData, setUSerData] = useState(null);
+  const [isModal, setIsModal] = useState(null);
+  const [littleTitle, setLittleTite] = useState(null);
+  const [isSelected, setIsSelected] = useState(null);
+  const [isKebab, setIsKebab] = useState(null);
+  const [buttons, setButtons] = useState();
+  const [cardData, setCardData] = useState();
+
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const makeSearchList = async () => {
+    const { data } = await getUserList();
+
+    const filteredSearch = data.filter(
+      (card) =>
+        card.title?.toLowerCase().includes(search?.toLowerCase()) ||
+        card.url?.toLowerCase().includes(search?.toLowerCase()) ||
+        card.description?.toLowerCase().includes(search?.toLowerCase())
+    );
```

Comment:
소소하게 아래처럼 추상화를 해도 좋겠네요

```suggestion
    const isInclude = (data, key = search) => data?.toLowerCase().includes(key?.toLowerCase())
    
    const filteredSearch = data.filter(
      (card) => isInclude(card.title) || isInclude(card.url) || isInclude(card.description)
    );
```